spec/lib/evss/disability_compensation_form/form526_to_lighthouse_transform_spec.rb

```diff
@@ -197,6 +201,23 @@
       result = transformer.send(:convert_approximate_date, JSON.parse({ month: '03', year: '1973' }.to_json))
       expect(result).to eq('1973-03')
     end
+
+    it 'sets the is related to toxic exposure flag when matching to TE conditions' do
+      toxic_exposure_conditions = submission.form['form526']['form526']['toxicExposure']['conditions']
+      results = transformer.send(:transform_disabilities, data, toxic_exposure_conditions)
```

Comment:
Could these repeated variables, `toxic_exposure_conditions` and `results`, instead be defined as memoized helper methods?